lib/oplog/tail.go

```diff
@@ -103,6 +103,14 @@ var (
 		Name:      "last_received_staleness",
 		Help:      "Gauge recording the difference between this server's clock and the timestamp on the last read oplog entry.",
 	}, []string{"ordinal"})
+
+	metricOplogResumeGap = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "otr",
+		Subsystem: "oplog",
+		Name:      "resume_gap_seconds",
+		Help:      "Histogram recording the gap in time that a tailing resume had to catchup and whether it was successful or not.",
+		Buckets:   []float64{1, 2.5, 5, 10, 25, 50, 100, 250, 500, 1000},
+	}, []string{"status"})
```

Comment:
I see the point in making this a string, but for now the value is more or a less a bool, correct?

Reply:
And.. the histogram buckets < `MaxCatchUp` should always be `successful`, and the buckets >= `MaxCatchUp` should always be failure? So that assuming you knew the max catch up value at a certain time and the historgram threshold, you would be able to derive the success/failure value?

Reply:
Yes, it could be derived from that, but those are tunable parameters so having the label I think makes sense, since it's explicitly stating the action taken.  Failures are noteworthy since it means downstream meteor instances will potentially be in an inconsistent state.

As for string/bool -- I could go either way -- I don't know that we'd expand it in the future or not, so maybe bool is enough.  @torywheelwright do you have thoughts on this?

Reply:
My understanding is that this is a literal array of strings declaring the various label names, rather than a declaration of the type of the value (which I understand is always a float).

It is true that if you exported the configured max resume time as a different metric, you could derive whether the resume was successful or not. This is probably a better encoding strictly speaking, though it makes the query a little more complicated. I have no strong preference.